src/entities/Address.ts

```diff
@@ -0,0 +1,32 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { User } from './User';
+
+@Entity()
+export class Address {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  cep: string;
+
+  @Column()
+  street: string;
+
+  @Column()
+  streetNum: number;
+
+  @Column({ nullable: true })
+  complement: string;
```

Comment:
```suggestion
  @Column({ nullable: true })
  complement?: string;
```

Legal acomapnhar a opcionalidade do campo no Typescript tbm.